### DISCUSSION TITLE
INCLUYAN A PERU PARA 2FA HDPTA

### DISCUSSION THREAD

**Discussion by djvamps** [community/community#142441] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

INCLUYAN A PERU PARA LA 2FA, NO SEAN CABROS LOS DE GITHUB

**Comment by queenofcorgis** [comment]
Hey there!  👋 

We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.

You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
